models/topics-model.js

```diff
@@ -0,0 +1,9 @@
+const connection = require('../db/connection');
+const topicData = require('../db/data/test-data/topics');
+
+const selectTopics = () => {
+    console.log(topicData);
+    return Promise.resolve(topicData)
```

Comment:
Here and for all the following endpoints we want to communicate with the database instead of just requiring a file. Please make use of psql here and query the database, then pass the data through to the controller.